pkg/mysql/mysql.go

```diff
@@ -483,6 +492,13 @@ func containers(cr *apiv1alpha1.PerconaServerMySQL, secret *corev1.Secret) []cor
 func mysqldContainer(cr *apiv1alpha1.PerconaServerMySQL) corev1.Container {
 	spec := cr.MySQLSpec()
 
+	// In OpenShift, we should use emptyDir for ./mysqlsh to avoid permission issues.
+	mysqlshMountPath := mysqlshMountPathDefault
+
+	if customPath, exists := os.LookupEnv(naming.EnvMysqlshUserConfigHome); exists && customPath != "" {
+		mysqlshMountPath = customPath
+	}
```

Comment:
i don't understand why do we do this. should user set `MYSQLSH_USER_CONFIG_HOME` env variable to operator?

Reply:
User can set it, if he want (as we discussed and you asked above), but if user don't set this value, we use default one. Could you please add more details how do you see we can fix it? <3